app/views/audits/audits/index.html.erb

```diff
@@ -8,4 +8,7 @@
 <% end %>
 
 <%= render "no_content_to_audit" if content_items.empty? %>
+<% if @first_item %>
+    <%= link_to "Start audit of first item", content_item_audit_path(@first_item), class: "btn btn-success", style: 'margin-bottom:10px;' %>
```

Comment:
This is not passing in the filter parameters so when user clicks to go to next item it is not the next item in the list

Reply:
Nice one! I have fixed it 👍